packages/synthetix-main/cannonfile.toml

```diff
@@ -37,8 +37,8 @@ modified = ["contracts/modules/esnx", "contracts/modules/common", "scripts"]
 [run.deploy_core]
 exec = "scripts/deploy.js"
 func = "deploy"
-args = ["Core", ".*/(core|common)/.*"]
-modified = ["contracts/modules/core", "contracts/modules/common", "contracts/utils", "scripts"]
+args = ["Core", ".*/(core|common|mocks)/.*"]
+modified = ["contracts/modules/core", "contracts/modules/common", "contracts/modules/mocks", "contracts/utils", "scripts"]
```

Comment:
How do we mock modules?

I found this pattern in the code, and I finished it to make the most minimal noise with previous work.

If we go this way, we might want to enable cannon to exclude folders in prod builds.